tests/test_input_model.py

```
@@ -0,0 +1,438 @@
+            "--input-model",
+            "pathlib:Path",
+        ])
+
+        assert exit_code == Exit.ERROR
+        captured = capsys.readouterr()
+        assert "is not a supported type" in captured.err
+
+    def test_pydantic_v1_model_error(
+        self,
+        pydantic_model_module: Path,
+        tmp_path: Path,
```

Ruff (0.14.10)

Unused method argument: `tmp_path`

(ARG002)